dev/com.ibm.websphere.appserver.features/visibility/auto/javaBatch-1.0/com.ibm.websphere.appserver.adminCenter.tool.javaBatch-1.0.feature

```diff
@@ -0,0 +1,15 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
Why is this auto-feature in a sub-folder? What is the benefit? Why not directly in
`dev/com.ibm.websphere.appserver.features/visibility/auto` ?

Reply:
Oh, I see... it's because of the additional NLS properties files. However, an auto-feature is private, it does not need any translation.

Reply:
It isn't just nls properties files, but also icons needed for the admin center that are stored in the features directory in liberty.

Reply:
Storing icons (anything) in the features directory is rather unusual? It certainly isn't done for any other OL feature. Why is this one different?